eslint.config.js

```diff
@@ -0,0 +1,21 @@
+const nodePlugin = require('eslint-plugin-n');
+const eslintPluginPrettierRecommended = require('eslint-plugin-prettier/recommended');
+
+module.exports = [
+  nodePlugin.configs['flat/recommended-script'],
+  eslintPluginPrettierRecommended,
+  {
+    plugins: {
+      n: nodePlugin,
+    },
+    rules: {
+      // Make an exception for ESLint-related modules.
```

Comment:
i'm iffy about this, but couldn't pass linting without these exceptions. the alternative would be to move eslint and co from dev-deps to deps, but that strikes me as wrong.
@jrjohnson any thoughts?